samples/snippets/src/test/java/com/example/spanner/admin/archived/PgSpannerSampleIT.java

```diff
@@ -130,7 +130,7 @@ public void testSample() throws Exception {
     assertThat(databaseId).isNotNull();
 
     System.out.println("Create Database ...");
-    String out = runSample("createpgdatabase");
+    String out = runSample("createdatabase");
```

Comment:
Are you sure? This sample is for creating a PG database, so I think that this was not a typo.